scripts/jest/utils/index.js

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+const glob = require('glob');
+const { default: diff } = require('jest-diff');
```

Comment:
Should `jest-diff` be explicitly added as a devDependency?

Reply:
I was able to get rid of the dependency on `jest-diff`. The `diff` method is in fact exposed via the `this` value.